support/support.go

```diff
@@ -33,6 +33,7 @@ var (
 	TestTimeoutShort           = 1 * time.Minute
 	TestTimeoutMedium          = 2 * time.Minute
 	TestTimeoutLong            = 5 * time.Minute
+	TestTimeoutDouble          = 10 * time.Minute
```

Comment:
Nitpick: Personally I prefer timeouts specified in tests themselves. It is has really a meaning you can keep it.